loadtest/client.go

```diff
@@ -90,14 +90,14 @@ func getAdminClient(httpClient *http.Client, serverURL string, adminEmail string
 		mlog.Info(fmt.Sprintf("Failed to login as admin user: %s", resp.Error.Error()))
 		if cmdrun == nil {
 			mlog.Error("Unable to create admin user because was not able to connect to app server. Please create the admin user manually or fill in SSH information.")
-			mlog.Error(fmt.Sprintf("Command to create admin user: ./bin/mattermost user create --email %v --password %v --system_admin --username ltadmin", adminEmail, adminPass))
+			mlog.Error(fmt.Sprintf("Command to create admin user: ./bin/mattermost user create --email %v --password %v --system_admin --username %v", adminEmail, adminPass, adminUsername))
 			return nil
 		}
 		mlog.Info("Attempting to create admin user.")
-		if success, output := cmdrun.RunPlatformCommand(fmt.Sprintf("user create --email %v --password %v --system_admin --username ltadmin", adminEmail, adminPass)); !success {
+		if success, output := cmdrun.RunPlatformCommand(fmt.Sprintf("user create --email %v --password %v --system_admin --username %v", adminEmail, adminPass, adminUsername)); !success {
 			mlog.Error("Failed to create admin user", mlog.String("output", output))
 		}
-		if success, output := cmdrun.RunPlatformCommand(fmt.Sprintf("user verify ltadmin")); !success {
+		if success, output := cmdrun.RunPlatformCommand(fmt.Sprintf("user verify %v", adminUsername)); !success {
```

Comment:
@crspeller, here's where this failed for a customer: trying to verify the hard-coded `ltadmin` instead of the actual username for the user in question. Is there a way to verify the user using the email instead?

Reply:
Yup. All the user command should take email or username. Just use email instead here and that should fix the issue.

Reply:
@crspeller I've reverted the changes and only fixed this line. Should be sufficient?